snippets/base/tests/test_commands.py

```diff
@@ -178,34 +180,58 @@ def test_base(self):
             status=models.Job.PUBLISHED,
             limit_impressions=10000,
             metric_impressions=10000,
+            metric_last_update=now,
             publish_end=datetime.utcnow() + timedelta(days=1))
         job_click_limit_reached = JobFactory(
             status=models.Job.PUBLISHED,
             limit_impressions=10000,
             metric_impressions=1000,
             limit_clicks=1000,
             metric_clicks=1001,
+            metric_last_update=now,
             publish_end=datetime.utcnow() + timedelta(days=1))
         job_block_limit_reached = JobFactory(
             status=models.Job.PUBLISHED,
             limit_blocks=10,
             metric_blocks=1000,
+            metric_last_update=now - timedelta(minutes=30),
             publish_end=datetime.utcnow() + timedelta(days=1)
         )
         job_impression_limit_not_reached = JobFactory(
             status=models.Job.PUBLISHED,
             limit_impressions=100,
             metric_impressions=10,
+            metric_last_update=now - timedelta(hours=12),
             publish_end=datetime.utcnow() + timedelta(days=1))
         job_click_limit_not_reached = JobFactory(
             status=models.Job.PUBLISHED,
             limit_clicks=100,
             metric_clicks=10,
+            metric_last_update=now - timedelta(hours=1),
             publish_end=datetime.utcnow() + timedelta(days=1))
         job_block_limit_not_reached = JobFactory(
             status=models.Job.PUBLISHED,
             limit_blocks=100,
             metric_blocks=10,
+            metric_last_update=now,
+            publish_end=datetime.utcnow() + timedelta(days=1))
```

Comment:
NIT: I'm sure it's just copypasta, but you're using the `now` variable created at the top of the method in some places and calling `datetime.utcnow()` again in others.

Reply:
Thanks!